### PR TITLE
adding timestamp and address fields to many events

### PR DESCRIPTION
**Pull request by brossetti1**
## GitHub Issue

fixes https://github.com/HausDAO/daohaus-monorepo/issues/238

## Changes

adding several timestamp and address details to Dao, Proposals, Votes, and Member

includes build files

## Packages Added

N/A

## Checks

Before making your PR, please check the following:

- [X] Critical lint errors are resolved
- [X] App runs locally
- [X] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

**Comment by brossetti1** [comment]
@skuhlmann i noticed Dao already had `transactionHashSummon` so skipped `summonTxHash`

wondering if we need to update fields in:

transformProposal : https://github.com/HausDAO/daohaus-monorepo/blob/develop/libs/dao-data/src/utils/transformers.ts#L15
transformMembershipList: https://github.com/HausDAO/daohaus-monorepo/blob/develop/libs/dao-data/src/utils/transformers.ts#L56

File: apps/v3-subgraph/schema.graphql [hunk 1]
```diff
@@ -103,10 +107,26 @@ type Proposal @entity {
   sponsored: Boolean!
   "address that sponsored the proposal"
   sponsor: Bytes!
+  "transaction hash of the proposal sponsor"
+  proposalSponsorTxHash: Bytes!
```
Comment (skuhlmann) [inline apps/v3-subgraph/schema.graphql:111]: This should probably be nullable as we won't have data for that field when saving the proposal upon submission.

File: apps/v3-subgraph/schema.graphql [hunk 2]
```diff
@@ -103,10 +107,26 @@ type Proposal @entity {
...
+  "unix timestamp of when the proposal was sponsored"
+  proposalSponsorTxAt: BigInt!
```
Comment (skuhlmann) [inline apps/v3-subgraph/schema.graphql:113]: nullable

File: apps/v3-subgraph/schema.graphql [hunk 3]
```diff
@@ -103,10 +107,26 @@ type Proposal @entity {
...
   "indicates if the proposal is cancelled"
   cancelled: Boolean!
+  "transaction hash of the cancelled proposal"
+  proposalCancelTxHash: Bytes!
```
Comment (skuhlmann) [inline apps/v3-subgraph/schema.graphql:117]: same with these - all the proposal states that might not happen or don't happen until later in the proposal lifecycle will need to be nullable.

sponsor
cancelled
processed

File: apps/v3-subgraph/schema.graphql [hunk 4]
```diff
@@ -194,6 +218,8 @@ type Member @entity {
   delegatingTo: Bytes!
   "subgraph id of member the address is delegating to"
   delegatingToMember: Member
+  "the transaction hash when the delegate was last updated"
+  lastDelegateUpdateTxHash: Bytes!
```
Comment (skuhlmann) [inline apps/v3-subgraph/schema.graphql:222]: nullable

**Review by skuhlmann — CHANGES_REQUESTED** [review]

**Review by skuhlmann — APPROVED** [review]

**Pull request merged**